eth/chains/base.py

```diff
@@ -592,7 +610,8 @@ def get_transaction_result(
         This is referred to as a `call()` in web3.
         """
         with self.get_vm(at_header).state_in_temp_block() as state:
-            computation = state.costless_execute_transaction(transaction)
+            # Ignore is to not bleed the SpoofTransaction deeper into the code base
+            computation = state.costless_execute_transaction(transaction)  # type: ignore
```

Comment:
Probably a better solution here is an abstract interface for the transaction that `SpoofTransaction` and the main RLP Transaction both implement.

Reply:
That ignore is gone in the final version that was merged.